src/main/java/com/wavefront/dropwizard/metrics/DropwizardMetricsReporter.java

```diff
@@ -309,26 +332,32 @@ public void report(SortedMap<String, Gauge> gauges,
       for (Map.Entry<String, Gauge> entry : gauges.entrySet()) {
         if (entry.getValue().getValue() instanceof Number) {
           reportGauge(entry.getKey(), entry.getValue());
+          gaugesReported.inc();
         }
       }
 
       for (Map.Entry<String, Counter> entry : counters.entrySet()) {
         reportCounter(entry.getKey(), entry.getValue());
+        countersReported.inc();
       }
 
       for (Map.Entry<String, Histogram> entry : histograms.entrySet()) {
         reportHistogram(entry.getKey(), entry.getValue());
+        histogramsReported.inc();
```

Comment:
ditto
break this into WavefrontHistogramReported and nonWavefrontHistogramReported